security-analysis/security-analysis-api/src/main/java/com/powsybl/security/monitor/StateMonitor.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security.monitor;
+
+import java.util.*;
+
+/**
+ * @author Etienne Lesot <etienne.lesot at rte-france.com>
+ *
+ *
+ * A stateMonitor allows to get informations on the network after a loadflow computation
```

Comment:
Small typo, `information` is invariable.
Maybe specify the information is linked to the contingencies specified in the context